From 89a33a2d52087def8e8e79a441ce5cfa0bbd8f87 Mon Sep 17 00:00:00 2001
From: Daniele Tricoli <eriol@mornie.org>
Date: Thu, 8 Oct 2015 13:41:42 -0700
Subject: Use the bundle provided by ca-certificates instead of

 the embedded one.
Forwarded: not-needed
Last-Update: 2014-10-08

Patch-Name: 01_use-system-ca-certificates.patch
---
 MANIFEST.in       | 2 +-
 requests/certs.py | 4 ++--
 setup.py          | 2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/MANIFEST.in b/MANIFEST.in
index 439de49..7888aee 100644
--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1 +1 @@
-include README.rst LICENSE NOTICE HISTORY.rst test_requests.py requirements.txt requests/cacert.pem
+include README.rst LICENSE NOTICE HISTORY.rst test_requests.py requirements.txt
diff --git a/requests/certs.py b/requests/certs.py
index f922b99..248d12f 100644
--- a/requests/certs.py
+++ b/requests/certs.py
@@ -18,8 +18,8 @@ try:
 except ImportError:
     def where():
         """Return the preferred certificate bundle."""
-        # vendored bundle inside Requests
-        return os.path.join(os.path.dirname(__file__), 'cacert.pem')
+        # On Debian systems use ca-certificates
+        return '/etc/ssl/certs/ca-certificates.crt'
 
 if __name__ == '__main__':
     print(where())
diff --git a/setup.py b/setup.py
index c240624..1fc2a6e 100755
--- a/setup.py
+++ b/setup.py
@@ -70,7 +70,7 @@ setup(
     author_email='me@kennethreitz.com',
     url='http://python-requests.org',
     packages=packages,
-    package_data={'': ['LICENSE', 'NOTICE'], 'requests': ['*.pem']},
+    package_data={'': ['LICENSE', 'NOTICE'],},
     package_dir={'requests': 'requests'},
     include_package_data=True,
     install_requires=requires,
